README.md

```diff
@@ -1 +1,57 @@
-# ci-script
\ No newline at end of file
+# ci-script
+
+Script to help setup a repo with continuous integration support.
+
+- Uses GitHub Actions.
+- Actions configs for different setups.
+
+## Usage
+
+From your source dir, run:
+
+    ../ci-script/setup-ci MODE
+
+Where MODE is one of:
+
+- bedrock
+- bedrock-web
+- isomorphic
+- lint
+
+## Lint
+
+If you are using lint support:
+
+    npm i -D eslint
+    npm i -D eslint-config-digitalbazaar
+    # optional: eslint-plugin-jsdoc
+    # optional: eslint-plugin-vue
+    # setup a .eslintrc.js
+
+Add a `"script"` target something like:
+
+    "lint": "eslint ."
+
+## Coverage
+
+If you are using coverage support:
+
+    npm i -D nyc
+
+Add a `"script"` target something like:
+
+    "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=lcovonly npm run test-node",
+
+Note the `lcovonly` reporter. The default setup will use `./coverage/lcov.info`.
+
+Ensure package.json or nyc config is setup something like:
+
+    "nyc": {
+      "exclude": [
+        "tests"
+      ],
+      "reporter": [
+        "html",
+        "text-summary"
+      ]
```

Comment:
I'm not sure this is necessary.  What does it do?

Reply:
This is unfortunately something that's been copy & pasted everywhere.  I'm not sure what people want.  These are just used when no nyc --reporter options are given.  If there are *lots* of files, then the default "text" is a bit much.  Long ago I figured using lcov + text-summary for "npm run coverage" would create the pretty html and short output.  And these options were not here so "npm run coverage-report" (that just calls report) would use the default "text" reporter with all the per-file info.  But that got all lost in history.  What should the recommended approach be now?

Reply:
I think we bake the reporters we need into the npm scripts and avoid this extra boilerplate.

Also, does the `coverage-report` script in some modules do anything that the regular report (the one we're running for CI) does not do? 

What we want here is simply that the CI reports need to continue working, and we need a way for devs to generate/view an HTML coverage report locally.